app/src/main/java/org/p2p/wallet/user/interactor/UserInteractor.kt

```diff
@@ -154,7 +154,7 @@ class UserInteractor(
             val price = prices.find { it.tokenId == token.coingeckoId }
             val totalInUsd = price?.price?.let { token.total.times(it) }
             token.copy(rate = price?.price, totalInUsd = totalInUsd)
-        }
+        }.sortedWith(TokenComparator())
```

Comment:
We are sorting before saving into DB. 
Are you sure the `map` here breaks the sorting?